tests/validation/tests/v3_api/test_create_ha.py

```diff
@@ -316,6 +316,8 @@ def set_route53_with_ingress():
         return
     elif RANCHER_LOCAL_CLUSTER_TYPE == "K3S":
         return
+    elif RANCHER_LOCAL_CLUSTER_TYPE == "RKE2":
```

Comment:
nit-pick: Can we make it like `elif RANCHER_LOCAL_CLUSTER_TYPE in ("RKE", "RKE2", "K3S"): return` since the conditions are doing the same thing? CC: @rancher-max ?

Reply:
Updated and squashed commit. Nice suggestion :)